encoders/numeric/RandomGaussianEncoder/tests/test_randomgaussianencoderload.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import pickle
+
+from .. import RandomGaussianEncoder
+from .. import TransformEncoder
+
+
+class RandomGaussianEncoderLoadTestCase:
+    def test_randomgaussinencoderload(self):
```

Comment:
be aware that these tests are not testing anything. a `test` does not have to return anything, but assert conditions. You have to check the `NumericTestCase` it was inheriting from and see what was really being tested there